README.md

```diff
@@ -1,18 +1,44 @@
-# AFrame & Redux - What This Is
+# A-Frame + Redux
 
-At [Olio Apps](http://www.olioapps.com/), we are applying software engineering idioms from React and React Native development ecosystems to WebVR development.  
+At [Olio Apps](http://www.olioapps.com/), we are applying tools and practices from our experience creating React and React Native applications to engineering VR software for the web.  
 
-This project is our effort to make a development platform that can be used to create aframe components and subclasses which makes it easy to connect your VR components to a redux store, for these reasons:
+## Installation 
 
-- to facilitate the use of data driven logic.
-- to be able to manage state the same way as other front end javascript/redux applications, and also to be able to think about the components in a familiar React-esque way.
-- to facililate easier onboarding for new and more experienced developers for rapid development and component creation.
+### npm
+```javascript
+npm install --save aframe-typescript-redux
+// or yarn add aframe-typescript-redux
+```
+```javascript
+import * as AframeRedux from "aframe-typescript-redux"
+```
 
-We will be discussing the [complete example](examples/connected_component.html) below. You can also play with a live example in action at [https://codesandbox.io/s/o71qm45xy](https://codesandbox.io/s/o71qm45xy).
+### cdn
 
-[![Foo](./docs/preview.png)](https://codesandbox.io/s/o71qm45xy)
+```html
+<!-- index.html -->
+...
+    <script 
+        type="text/javascript"
+        src="https://cdn.jsdelivr.net/npm/aframe-typescript-redux@0.0.7/dist-umd/vendor.bundle.min.js">
+    </script>
+    <script
+        type="text/javascript"
+        src="https://cdn.jsdelivr.net/npm/aframe-typescript-redux@0.0.7/dist-umd/index.min.js">
+    </script>
+</head>
+```
 
-## OVERVIEW OF CONNECTING AN AFRAME COMPONENT TO A REDUX STORE
+### local development
+```
+git clone https://github.com/aframevr/aframe.git
```

Comment:
at this stage, are we trying to show how people can extend the various framework pieces in here?  -- or are we trying to show them how to setup a local environment for using the framework pieces for creating their own subclasses for their projects ...? or are we trying to show them how to create an aframe project in general?

i think we should be clearer about the objective of the local development section if we have this in here